### PR TITLE
{devel}[GCCcore/12.2.0] flatbuffers v23.1.4

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-23.1.4-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeNinja'
```
Comment (verdurin) [inline easybuild/easyconfigs/f/flatbuffers/flatbuffers-23.1.4-GCCcore-12.2.0.eb:1]: @SimonPinches please reinstate the previous author attribution

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-23.1.4-GCCcore-12.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/f/flatbuffers/flatbuffers-23.1.4-GCCcore-12.2.0.eb:4]: Done - apologies.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3155.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/0e8475f36bfeaf3e9d1fdf5708087763 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/cc7f69d28643b86fed3b1c431a56b5a9 for a full test report.

**Pull request merged**